tests/test_tracing_tracer.py

```diff
@@ -112,6 +112,37 @@ def test_tracer_start_traced_span_with_headers(tracer):
     assert tracer.logger.debug.called_with_args('Span started test_current_span')
 
 
+@patch(MODULE_PATH + 'generate_identifier', lambda n: f'test_generated_id_{n}')
+@patch(MODULE_PATH + 'get_timestamp', MagicMock(return_value='test_timestamp'))
+@patch(CLASS_PATH + 'current_span', 'test_current_span')
+def test_tracer_start_traced_span_with_gcp_loadbalancer_headers(tracer):
+    tracer.current_span = ''
+
+    test_gcp_span_headers = {
+        "X-Cloud-Trace-Context": "test_gcp_trace_id/test_gcp_span_id;params"
+    }
+
+    tracer.start_traced_span(test_gcp_span_headers, 'test_span_name')
+
+    expected_spans = {
+        'test_gcp_span_id': {
+            "start_timestamp": 'test_timestamp',
+            "display_name": 'test_service_name:test_span_name',
+            "child_span_count": 0,
+            "values": {
+                'X-B3-SpanId': 'test_gcp_span_id',
+                'X-B3-TraceId': 'test_gcp_trace_id',
+                'X-B3-Flags': None,
+                'X-B3-ParentSpanId': None,
+                'X-B3-Sampled': None
+            }
+        }
+    }
+    assert tracer._spans == expected_spans
+    assert tracer.memory.current_span_id == 'test_gcp_span_id'
+    assert tracer.logger.debug.called_with_args('Span started test_current_span')
+
+
```

Comment:
It would be good to have some tests for different examples of the `X-Cloud-Trace-Context` header (e.g. with/without options, when it's malformed (missing the `/`)).